### DISCUSSION TITLE
Sideloading is the only option, if do not want to go to the store?

### DISCUSSION THREAD

**Discussion by kazutsugu** [dotnet/maui#7294] [Q&A] [answered]
We are considering MAUI (actually Xamarin UWP at this moment, and migrate to MAUI in future) for our Windows app for public. But for some reason, we do not want to go to the store.

According to our research, it appears that it is obligatory to go thru the store. Can any of you confirm that it is correct?
That it is the case today with Xamarin UWP, and MAUI keeps the same distribution model?

Signing with a commercial certificate is not an issue.
However, forcing users to enable sideload mode is out of question.
If the above is the case, we need to drop Xamarin and MAUI.

Many thanks in advance for your advice.
Kazu

**Comment by JosHuybrighs** [comment, marked as answer]
@kazutsugu  I am not sure why sideloading would not be an option for you in 2022. All activities in the area of security and making sure customers get a trust-worthy package in today's Windows, focus on either deployment through the store or side-loading. I also think (but can be wrong) that with side-loading you can still make the app available on your server.
Now, it is not clear for me why you want to use maui if the only target is Windows. I think the Windows App SDK with winui3 (e.g. in a desktop app) will offer might more capabilities.

**Reply by kazutsugu** [reply]
@JosHuybrighs Thank you for your response. Actually, I asked this question at multiple places, and got a clear answer at one of those channels. https://docs.microsoft.com/en-us/windows/application-management/sideload-apps-in-windows-10 says: _Starting with Windows 10 2004, sideloading is enabled by default. You can deploy a signed package onto a device without a special configuration_. I believe that you too are well aware of this change, and that is why you are saying "_in 2022_". The Win10 2004 was out in the year 2020. So by now, majority of ordinary users would not even know that it is a sideloaded package. The PC where I tested this happens to be "before 2004" Win10, and got the enable sideload mode, which needs the local admin privilege, prompt.
Why Xamarin, and MAUI? We are producing mobile apps as well, and for which chosen the framework. Therefore, aiming for a high productivity and stable and good quality products, we think that sticking to a single skill set for both desktop and mobile would be advantageous. Hope this clarifies.
